spec/changelog.yml

```diff
@@ -2,17 +2,17 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom of each in-development version.
 ##
-- version: 2.8.1-next
+- version: 2.9.0-next
   changes:
-  - description: Prepare for next version
-    type: enhancement
-    link: https://github.com/elastic/package-spec/pull/522
   - description: Add validation for data types of metrics
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/525
   - description: Extend benchmarks spec to add rally scenario
     type: enhancement
     link: https://github.com/elastic/package-spec/pull/524
+  - description: Add support to define routing rules in data streams (technical preview)
+    type: enhancement
+    link: https://github.com/elastic/package-spec/pull/535
```

Comment:
Added `technical preview` since it is documented also that `reroute processors` is in technical preview too. https://www.elastic.co/guide/en/elasticsearch/reference/current/reroute-processor.html

Is that ok ? @jsoriano @juliaElastic